deployments/optimism-goerli/usdc/configuration.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x7E07E15D2a87A24492740D16f5bdF58c16db0c4E",
+  "baseTokenPriceFeed": "0x2636B223652d388721A0ED2861792DA9062D8C73",
+  "borrowMin": "100e6",
+  "pauseGuardian": "0xe7661C55fe281f986FA5312D57d19ADb6064462F",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "1000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000011574074074074073e15",
+    "baseBorrowSpeed": "0.0011458333333333333e15",
+    "baseMinForRewards": "10000e6"
+  },
+  "rewardTokenAddress": "0x6AF3cb766D6cd37449bfD321D961A61B0515c1BC",
+  "assets": {
+    "OP": {
+      "priceFeed": "0xE882831E58eec48B7f304482771F67e6b846733D",
```

Comment:
Trying to populating infos into c3-api to start supporting optimism. But it seems we're missing OP's address here 🤔 ? Is it '0x4200000000000000000000000000000000000042' ? 
(Just checked on testnet, it seems it's also this address, but wanted to double check :) )

Reply:
Yes "0x4200000000000000000000000000000000000042" is the official address for OP on both Optimism testnet and mainnet. However, on testnet, we've deployed our own OP token because there doesn't seem to be a public faucet for it and it would make using the testnet easier if we controlled the token supply. Our testnet OP token address is "0x32719c5e2e35909C8f42167315445e349e749E05".

Btw, if you are unsure which addresses we use on each chain for each deployment, you can use Spider to programmatically pull in all the relevant addresses for a deployment. See https://github.com/compound-finance/comet/tree/main#spider

Reply:
Got it thank you so much!! 🙏